app/lib/bugs_import.rb

```diff
@@ -0,0 +1,23 @@
+require 'csv'
+
+class BugsImport
+  attr_reader :url
+
+  def initialize(url)
+    @url = url
+  end
+
+  def execute
+    ActiveRecord::Base.transaction do
+      Bug.delete_all
+
+        CSV.parse(data, headers: true) do |row|
+          Bug.create!(row.to_hash)
+      end
```

Comment:
`end` at 16, 6 is not aligned with `CSV.parse(data, headers: true) do |row|` at 14, 8